tutorials/CMakeLists.txt

```diff
@@ -764,7 +764,7 @@ if(ROOT_pyroot_FOUND)
   endif()
 
   find_python_module(keras QUIET)
```

Comment:
adding tmva-pymva is not really needed if we search for work or eras when tmva-pymva is ON

Reply:
Right, thanks for the catch! I will remove that commit